### ISSUE TITLE
Feature request: add current box version to the "vagrant up" output

### ISSUE THREAD

**Issue by RohanTalip** [hashicorp/vagrant#10339]
### Vagrant version
2.1.2 due to issue #10338 

### Host operating system
macOS High Sierra 10.13.6

### Guest operating system
ubuntu/bionic64

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/bionic64"
end
```

### Debug output
n/a

### Expected behavior
`vagrant up` shows the box version, even if there is no update.

### Actual behavior
`vagrant up` only shows the box version if there is an update.

### Steps to reproduce
1. Run `vagrant up`

This would have helped to expose issue #10338 sooner.

**Comment by johanhammar** [comment]
Perhaps something like this?

```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'hashicorp/precise32' running version '1.0.0' is up to date...
```

But it might be a bit verbose when there is an update

```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' running version '20181002.0.0' is up to date...
==> default: A newer version of the box 'ubuntu/trusty64' for provider 'virtualbox' is
==> default: available! You currently have version '20181002.0.0'. The latest is version
==> default: '20181022.0.0'. Run `vagrant box update` to update.
```

I can provide a small PR if this change is wanted/approved?

**Comment by RohanTalip** [comment]
@johanhammar, that could work.  Would this also work for `vagrant box outdated`?

I agree that it could be a little verbose if there is an update, but personally I favour some or more information over no or less information.

An alternative (for `vagrant up`) could be:
```
$ vagrant up
Bringing machine 'default' up with 'ubuntu/bionic64 (virtualbox, 20181024.0.0)' ...
...
```

i.e. That way it would be easier to match against the output from `vagrant box list`:
```
$ vagrant box list
ubuntu/bionic64 (virtualbox, 20181018.0.0)
ubuntu/bionic64 (virtualbox, 20181024.0.0)
```

Yet another alternative could be to show a message prompting the user to update if there is a newer version (as is the case now), otherwise show the current version and indicate that it is up to date, for example:
```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'ubuntu/bionic64' version '20181024.0.0' is up to date.
```

... or:
```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'ubuntu/bionic64 (virtualbox, 20181024.0.0)' is up to date.
```

If necessary, these changes could be triggered by a command line argument (e.g. `-v` or `-i`), in case anyone has scripts depending on the current output format.
